hive3/src/main/java/org/apache/iceberg/mr/hive/vector/HiveVectorizedReader.java

```diff
@@ -21,7 +21,6 @@
 import java.io.IOException;
 import java.util.List;
 import java.util.Map;
-import org.apache.commons.lang3.ArrayUtils;
```

Comment:
should compilation not fail here?

Reply:
I did not removed the libraries from Gradle for the subprojects ( Maven expert but not so much on Gradle), I have been searching but seems that the sub-projects were using the parent declared libraries when needed... 

Will fix this line on next re-push and if you can point me to where I can delete the libraries for subprojects will be great!

Reply:
seems to be used here: https://github.com/apache/iceberg/blob/c07f2aabc0a1d02f068ecf1514d2479c0fbdd3b0/hive3/src/main/java/org/apache/iceberg/mr/hive/vector/HiveVectorizedReader.java#L104

Reply:
Have changed that code to native Java 8 compliant coe